qovery/data_source_helm.go

```diff
@@ -274,6 +280,196 @@ func (d helmDataSource) Schema(_ context.Context, _ datasource.SchemaRequest, re
+				Description: "Specify if the environment preview option is activated or not for this helm.",
+				Optional:    true,
+				Computed:    true,
+			},
+			"auto_deploy": schema.BoolAttribute{
+				Description: " Specify if the service will be automatically updated on every new commit on the branch.",
+				Optional:    true,
+				Computed:    true,
+			},
+			"arguments": schema.SetAttribute{
+				Description: "Helm arguments",
+				ElementType: types.StringType,
+				Optional:    true,
+				Computed:    true,
+			},
+			"allow_cluster_wide_resources": schema.BoolAttribute{
+				Description: "Allow this chart to deploy resources outside of this environment namespace (including CRDs or non-namespaced resources)",
+				Computed:    true,
+			},
+			"source": schema.SingleNestedAttribute{
+				Description: "Helm chart from a Helm repository or from a git repository",
+				Computed:    true,
+				Attributes: map[string]schema.Attribute{
+					"helm_repository": schema.SingleNestedAttribute{
+						Description: "Helm repositories can be private or public",
+						Optional:    true,
+						Attributes: map[string]schema.Attribute{
+							"helm_repository_id": schema.StringAttribute{
+								Description: "helm repository id",
+								Required:    true,
+							},
+							"chart_name": schema.StringAttribute{
+								Description: "Chart name",
+								Required:    true,
+							},
+							"chart_version": schema.StringAttribute{
+								Description: "Chart version",
+								Required:    true,
+							},
+						},
+					},
+					"git_repository": schema.SingleNestedAttribute{
+						Description: "Git repository",
+						Optional:    true,
+						Attributes: map[string]schema.Attribute{
+							"url": schema.StringAttribute{
+								Description: "Helm's source git repository URL",
+								Required:    true,
+							},
+							"branch": schema.StringAttribute{
+								Description: "Helm's source git repository branch",
+								Optional:    true,
+								Computed:    true,
+							},
+							"root_path": schema.StringAttribute{
+								Description: "Helm's source git repository root path",
+								Optional:    true,
+								Computed:    true,
+							},
+							"git_token_id": schema.StringAttribute{
+								Description: "The git token ID to be used",
+								Optional:    true,
+								Computed:    true,
+							},
+						},
+					},
+				},
+			},
+			"values_override": schema.SingleNestedAttribute{
+				Description: "Define your own overrides to customize the helm chart behaviour.",
+				Computed:    true,
+				Attributes: map[string]schema.Attribute{
+					"set": schema.MapAttribute{
+						ElementType: types.StringType,
+						Optional:    true,
+					},
+					"set_string": schema.MapAttribute{
+						ElementType: types.StringType,
+						Optional:    true,
+					},
+					"set_json": schema.MapAttribute{
+						ElementType: types.StringType,
+						Optional:    true,
+					},
+					"file": schema.SingleNestedAttribute{
```

Comment:
_suggestion_: Remove extra `the` => `Define overrides by selecting a YAML file from a git repository (preferred) or by passing raw YAML files.`

Reply:
done